modules/installing-rhv-example-install-config-yaml.adoc

```diff
@@ -70,8 +70,27 @@ platform:
 publish: External
 pullSecret: '{"auths": ...}'
 sshKey: ssh-ed12345 AAAA...
+compute:
+  platform:
+    ovirt:
+      ..
+      sparse: false # add this line
```

Comment:
Does call out item 1 relate to this line too? If so, you could add `<1>` to the end of this line.